enacts/enactstozarr.py

```diff
@@ -48,53 +36,45 @@ def set_up_dims(xda):
     # and look into xESMF.
     #
     # [1] https://climatedataguide.ucar.edu/climate-data-tools-and-analysis/regridding-overview
-    rr_mrg = rr_mrg.interp(
-        X=np.arange(rr_mrg.X.min(), rr_mrg.X.max() + RESOLUTION, RESOLUTION),
-        Y=np.arange(rr_mrg.Y.min(), rr_mrg.Y.max() + RESOLUTION, RESOLUTION),
-    )
-
-rr_mrg = rr_mrg.chunk(chunks=CHUNKS)
-
-xr.Dataset().merge(rr_mrg).to_zarr(
-  store = RR_MRG_ZARR_PATH
-)
-
-if not os.access(CONFIG["data_dir"], os.W_OK | os.X_OK):
-    sys.exit("can't write to output directory")
-
-def set_up_dims(xda):
-    datestr = Path(xda.encoding["source"]).name.split("_")[2]
-    year = int(datestr[0:4])
-    month = int(datestr[4:6])
-    dekad = int(datestr[6:7])
-    day = (dekad - 1) * 10 + 1
-    xda = xda.expand_dims(T = [dt.datetime(year, month, day)])
-    xda = xda.rename({'Lon': 'X','Lat': 'Y'})
-    return xda
+        print("Your data will be regridded. Refer to function documentation for more information on this.")
```

Comment:
@aaron-kaplan should we use raise warnings for this kind of stuff? not sure what that entails.

Reply:
If we go with the suggestion I made [elsewhere](https://github.com/iridl/python-maprooms/pull/277/files/3251997db01508771fb8000718d78c54db9c284b#r1091296468) , `resolution` will be null unless the config author intended to regrid it. In that case I don't think a warning is necessary.